spec/models/tags/api_tag_spec.rb

```diff
@@ -7,7 +7,8 @@
       name: "Weather of Boston Api",
       url: "www.supercool.com/something123456789",
       paid: true,
-      description: "This is what you want."
+      description: "This is what you want.",
+      user_id: 1
     )
     test_tag = Tag.create(name: "image_recognition")
     ApiTag.create(api: test_api, tag: test_tag)
```

Comment:
Factories with specific traits (apis with and without tags, for example) would greatly simplify the setup here!